src/api/app/models/distribution.rb

```diff
@@ -38,6 +38,10 @@ def update_from_xmlhash(xmlhash)
 
     self
   end
+
+  def variants
+    Distribution.where(vendor: vendor, reponame: reponame).where.not(id: id)
```

Comment:
This yells query object to me. It's for sure not an instance method.

Reply:
Hm but it depends on the instance. Only instances have variants right?

Reply:
Anyway it's not used at all. I can remove it if you don't like it.

Reply:
Let's remove it :)